sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/TestMethodShouldContainAssertion.Xunit.cs

```diff
@@ -191,11 +191,6 @@ public void ShouldSomething()
             Assert.IsTrue(true);
         }
 
-        public void AssertSomething()
```

Comment:
Miscellaneous: the code in the existing Xunit test class didn't compile due to a duplicate method definition for this method.